doc/examples/filters/plot_hysteresis.py

```diff
@@ -0,0 +1,50 @@
+"""
+=======================
+Hysteresis thresholding
+=======================
+
+*Hysteresis* is the lagging of an effect — a kind of inertia. In the
+context of thresholding, it means that areas above some *low* threshold
+are considered to be above the threshold *if* they are also connected
+to areas above a higher, more stringent, threshold. They can thus be
+seen as continuations of these high-confidence areas.
+
+Below, we compare normal thresholding to hysteresis thresholding.
+Notice how hysteresis allows one to ignore "noise" outside of the coin
+edges.
+"""
+
+import matplotlib.pyplot as plt
+from skimage import data, filters
+
+fig, ax = plt.subplots(nrows=2, ncols=2,
+                       subplot_kw={'adjustable': 'box-forced'})
+
+image = data.coins()
+edges = filters.sobel(image)
+
+low = 0.05
```

Comment:
Using '0.1' does miss part of the edge on a couple.  It cannot be above '0.07' or so to keep the edges intact.  Or perhaps I am misunderstanding the point of this thresholding.  Is the ideal result that only the edge of each coin would show up or is it desired to preserve the edges of interior features such as the person in the top left coin?